Lib/unittest/loader.py

```diff
@@ -94,6 +94,10 @@ def loadTestsFromTestCase(self, testCaseClass):
     def loadTestsFromModule(self, module, *, pattern=None):
         """Return a suite of all test cases contained in the given module"""
         tests = []
+        if pattern is not None and not fnmatch(
+            os.path.basename(module.__file__), pattern):
```

Comment:
```suggestion
                os.path.basename(module.__file__), pattern):
```

Extra identation for better readability: body and condition do not visually merge anymore.